### PR TITLE
Use IP addresses from documentation ranges cf. rfc5737

### PR DESCRIPTION
**Pull request by SphinxKnight**
When doing some updates in the French section, I noticed that some IP addresses in the docs were not from reserved ranges.

### Description

I changed existing IP addresses to reserved IP addresses for documentation

### Motivation

https://datatracker.ietf.org/doc/html/rfc5737

### Additional details

I may have been too "strict" for some articles and also have voluntarily left some mentions of non-reserved IP addresses when I felt they deserved to be kept in the documentation due to their nature (ex. ping mozilla.org…)

Reviewers may use the following regexp `(\d{1,3})\.(\d{1,3})\.(\d{1,3})\.(\d{1,3})` to look for IPv4 addresses. I used `[0-9a-f]{1,4}:[0-9a-f]{1,4}:[0-9a-f]{1,4}:[0-9a-f]{1,4}:[0-9a-f]{1,4}:[0-9a-f]{1,4}:[0-9a-f]{1,4}:[0-9a-f]{1,4}` for IPv6 but the existing occurrences wee fine.

https://en.wikipedia.org/wiki/Reserved_IP_addresses may be friendlier than the RFC if needed

### CODE REVIEW

File: files/en-us/glossary/round_trip_time/index.md [hunk 1]
```diff
@@ -10,15 +10,15 @@ page-type: glossary-definition
...
-round-trip min/avg/max/stddev = 23.781/26.828/34.904/4.114 ms
+PING example.com(2001:db8:220:1:248:1893:25c8:1946 (2001:db8:220:1:248:1893:25c8:1946)) 56 data bytes
+64 bytes from 2001:db8:220:1:248:1893:25c8:1946 (2001:db8:220:1:248:1893:25c8:1946): icmp_seq=1 ttl=52 time=85.4 ms
+64 bytes from 2001:db8:220:1:248:1893:25c8:1946 (2001:db8:220:1:248:1893:25c8:1946): icmp_seq=2 ttl=52 time=87.0 ms
+64 bytes from 2001:db8:220:1:248:1893:25c8:1946 (2001:db8:220:1:248:1893:25c8:1946): icmp_seq=3 ttl=52 time=86.7 ms
+64 bytes from 2001:db8:220:1:248:1893:25c8:1946 (2001:db8:220:1:248:1893:25c8:1946): icmp_seq=4 ttl=52 time=86.4 ms
+^C
+--- example.com ping statistics ---
+4 packets transmitted, 4 received, 0% packet loss, time 3004ms
+rtt min/avg/max/mdev = 85.436/86.363/86.973/0.577 ms
```
Comment (bsmth) [inline files/en-us/glossary/round_trip_time/index.md:21]: Mine's running IPV4 by default, by the way. Usually not a big deal, but for the sake of this article, it's a little less noise, what do you reckon?

```bash
ping example.com
PING example.com (93.184.216.34): 56 data bytes
64 bytes from 93.184.216.34: icmp_seq=0 ttl=54 time=110.882 ms
64 bytes from 93.184.216.34: icmp_seq=1 ttl=54 time=109.616 ms
64 bytes from 93.184.216.34: icmp_seq=2 ttl=54 time=109.749 ms
64 bytes from 93.184.216.34: icmp_seq=3 ttl=54 time=110.241 ms
^C
--- example.com ping statistics ---
4 packets transmitted, 4 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 109.616/110.122/110.882/0.497 ms
```

Comment (bsmth) [reply files/en-us/glossary/round_trip_time/index.md:21]: (replacing `93.184.216.34` with `203.0.113.0` or something?)

File: files/en-us/learn/common_questions/web_mechanics/what_is_a_domain_name/index.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ page-type: learn-faq
 
 Domain names are a key part of the Internet infrastructure. They provide a human-readable address for any web server available on the Internet.
 
-Any Internet-connected computer can be reached through a public {{Glossary("IP Address")}}, either an IPv4 address (e.g. `173.194.121.32`) or an IPv6 address (e.g., `2027:0da8:8b73:0000:0000:8a2e:0370:1337`).
+Any Internet-connected computer can be reached through a public {{Glossary("IP Address")}}, either an IPv4 address (e.g. `192.0.2.172`) or an IPv6 address (e.g., `2001:db8:8b73:0000:0000:8a2e:0370:1337`).
```
Comment (bsmth) [inline files/en-us/learn/common_questions/web_mechanics/what_is_a_domain_name/index.md:37]: Q: what's the recommended range for IPV6?

**Review by bsmth — APPROVED** [review]
I have one (editorial) comment about using IPV4 instead of IPV6 for an example for readability, but everything else looks good, ty. Leaving a +1 here 👍🏻

File: files/en-us/learn/common_questions/web_mechanics/what_is_a_domain_name/index.md (hunk 2 above)
Comment (bsmth) [reply files/en-us/learn/common_questions/web_mechanics/what_is_a_domain_name/index.md:37]: Answering my own question here: https://datatracker.ietf.org/doc/html/rfc3849

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**